pattern/crossfader.c

```diff
@@ -18,7 +18,7 @@ void crossfader_init(struct crossfader * crossfader) {
     // Render targets
     glGenFramebuffersEXT(1, &crossfader->fb);
     glGenTextures(1, &crossfader->tex_output);
-    if((e = glGetError()) != GL_NO_ERROR) FAIL("OpenGL error: %s\n", gluErrorString(e));
+    if((e = glGetError()) != GL_NO_ERROR) FAIL("OpenGL error: %s\n", GLU_ERROR_STRING(e));
```

Comment:
Cool, I won't do so in this patch, but to be clear you're talking about macroizing the whole `if((e = glGetError()) != GL_NO_ERROR) FAIL("OpenGL error: %s\n", GLU_ERROR_STRING(e));` dance into something like `ASSERT_GL_NO_ERROR`, correct?